dists/bash-completions/scummvm

```diff
@@ -0,0 +1,86 @@
+_scummvm()
+{
+    local opt_name
+    _init_completion || return
+
+    # If it's in the middle of a long option, offer completions for that. I'm
+    # adding these as an example until we figure which ones we want.
+
+    if [[ ${cur} == --* ]]; then
+       COMPREPLY=($(compgen -W "--debugflags= --debug-channels-only" -- "${cur}"))
+
+       # If the match ends with "=" we want to continue typing from there
+       # without any space at the end. This seems a bit peculiar to me, because
+       # it will apply to multiple matches, but it's the way that was suggested
+       # to me.
+
+       if [[ ${COMPREPLY[@]} =~ "=" ]]; then
+           compopt -o nospace
+       fi
+       
+       return;
+    fi
+
+    # Handle command line options. Short options are converted into their long
+    # form, and for long options we extract the option being used. At the point
+    # where we can do tab completion on a long option, ${prev} will be "=" so
+    # we have to look further back.
+
+    case ${prev} in
+        -e)
+            opt_name=music-driver
+            ;;
+        =)
+            opt_name=${COMP_WORDS[COMP_CWORD-2]}
+            if [[ ${opt_name} != --* ]]; then
+                return;
+            fi
+            opt_name="${opt_name#--}"
+            ;;
+    esac
+
+    # If it's not an option, try matching a target from the scummvm.init file.
+
+    if [[ -z ${opt_name} ]] && [[ ${cur} != -* ]]; then
+        local scummvm_config
+
+        # Try our old, now downright ancient, configuration path
```

Comment:
Why are you trying the deprecated configuration path first? Shouldn't that be the last one?

Reply:
I think that's what backends/platform/sdl/posix/posix.cpp does. Mind you, on my computer XDG_CONFIG_HOME isn't set and I no longer have a ~/.scummvmrc so this bit needs more testing.